src/toil/batchSystems/parasol.py

```diff
@@ -346,3 +346,10 @@ def shutdown(self):
         for results in self.resultsFiles.values():
             os.remove(results)
         os.rmdir(self.parasolResultsDir)
+
+    @staticmethod
+    def supportsWorkerCleanup():
+        """
+        Indicates whether the batch system supports the cleanup of workers on shutdown
```

Comment:
No need to duplicate the overridden method's docstring (unless you promise to keep these copies consistent from now until all eternity).